KBH/3Day/K번째수.js

```diff
@@ -0,0 +1,13 @@
+function solution(array, commands) {
+    var answer = [];
```

Comment:
```suggestion
    const answer = [];
```
아직도 var가 쓰이는 곳이 있는 것 같아요 ㅎㅎ

Reply:
문제에만 집중하다보니 신경 못 썼던것 같습니다 let으로 바꾸겠습니다

Reply:
@KimBYH 코드 가이드를 작성해드렸어요! let보다도 우선순위가 높은게 const입니다 :) 
코드를 작성 할 때 작성 const가 최우선순위로 정의하시느게 좋아요. 그리고 값의 재할당이 필요하다면 그때 let으로 바꿔주시는게 좋습니다. 객체, 배열 같은 참조형의 경우 웬만하면 const입니다